invenio_users_resources/config.py

```diff
@@ -62,7 +62,7 @@
     ),
     "username": dict(
         title=_("Username"),
-        fields=["username", "-created"],
+        fields=["username.keyword", "-created"],
```

Comment:
Since, we've changed this field in v3 to analyzed text, we need to mention keyword here.